src/app/models/chat-group.ts

```diff
@@ -1,8 +1,16 @@
 import { firestore } from 'firebase';
+import { KeyMapI } from './shared';
+
+// export interface ChatGroupI {
+//   id?: string;
+//   name?: string;
+//   participantIds: string[];
+//   createdAt: firestore.FieldValue | firestore.Timestamp;
+// }
```

Comment:
```suggestion
```
This is not needed. I only left it for you guys to review the changes if needed.